test/idleness_test.go

```diff
@@ -408,17 +408,20 @@ func (s) TestChannelIdleness_Enabled_IdleTimeoutRacesWithRPCs(t *testing.T) {
 	// Verify that the ClientConn moves to READY.
 	ctx, cancel := context.WithTimeout(context.Background(), defaultTestTimeout)
 	defer cancel()
-	awaitState(ctx, t, cc, connectivity.Ready)
+	client := testgrpc.NewTestServiceClient(cc)
+	if _, err := client.EmptyCall(ctx, &testpb.Empty{}); err != nil {
+		t.Errorf("EmptyCall RPC failed: %v", err)
+	}
```

Comment:
Why not just delete this entirely?  We're going to perform 20 RPCs after this anyway.

Reply:
This acts as a sanity check to ensure that the channel is `READY` and can make RPCs before we get to our actual test. So, if for some reason, the channel doesn't become `READY`, then we will fail here and will have a better signal instead of failing down below wherein we might initially think the failure is because of a bad interaction between idle timeout firing and a new RPC being made.